src/main/java/nextstep/subway/fare/domain/policy/age/AgeFarePolicyType.java

```diff
@@ -0,0 +1,39 @@
+package nextstep.subway.fare.domain.policy.age;
+
+import java.util.Arrays;
+import nextstep.subway.fare.domain.policy.age.impl.AdultAgeFarePolicy;
+import nextstep.subway.fare.domain.policy.age.impl.ChildAgeFarePolicy;
+import nextstep.subway.fare.domain.policy.age.impl.FreeAgeFarePolicy;
+import nextstep.subway.fare.domain.policy.age.impl.TeenagerAgeFarePolicy;
+
+public enum AgeFarePolicyType {
+
+    FREE(FreeAgeFarePolicy.class),
+    ADULT(AdultAgeFarePolicy.class),
+    CHILD(ChildAgeFarePolicy.class),
+    Teenager(TeenagerAgeFarePolicy.class);
+
+    private final Class<? extends AgeFarePolicy> ageFarePolicy;
+    AgeFarePolicyType(Class<? extends AgeFarePolicy> ageFarePolicy) {
```

Comment:
```suggestion
    private final Class<? extends AgeFarePolicy> ageFarePolicy;
    
    AgeFarePolicyType(Class<? extends AgeFarePolicy> ageFarePolicy) {
```

공백을 추가하면 가독성이 올라갈 것 같아요!